pkg/entrypoint/entrypointer.go

```diff
@@ -43,6 +45,21 @@ const (
 	FailOnError     = "stopAndFail"
 )
 
+// ContextError context error type
+type ContextError string
+
+// Error implements error interface
+func (e ContextError) Error() string {
+	return string(e)
+}
+
+var (
+	// ErrContextDeadlineExceeded is the error returned when the context deadline is exceeded
+	ErrContextDeadlineExceeded = ContextError(context.DeadlineExceeded.Error())
+	// ErrContextCanceled is the error returned when the context is canceled
+	ErrContextCanceled = ContextError(context.Canceled.Error())
```

Comment:
I'm curious if these error types are necessary given that they just wrap errors from the context package?